lib/dpl/provider/elastic_beanstalk.rb

```diff
@@ -138,6 +142,40 @@ def create_app_version(s3_object)
         eb.create_application_version(options)
       end
 
+      # Wait until EB environment update finishes
+      def wait_until_deployed
+        success = true
```

Comment:
I had to change the logic a bit, because EB may send events like `info, error, error, info`, and then success flag would be set to `true, false, false, true`, so would be true despite the errors. So i've added errors counter instead. If there's 1 or more error events - build fails.